ydb/src/client_topic/client.rs

```diff
@@ -0,0 +1,92 @@
+use crate::client::TimeoutSettings;
+use crate::client_topic::list_types::{Consumer, MeteringMode, SupportedCodecs};
+use crate::errors;
+use crate::grpc_connection_manager::GrpcConnectionManager;
+use crate::grpc_wrapper::raw_topic_service::create_topic::RawCreateTopicRequest;
+use crate::grpc_wrapper::raw_topic_service::delete_topic::RawDropTopicRequest;
+use crate::YdbError::InternalError;
+use crate::{grpc_wrapper, YdbResult};
+use derive_builder::{Builder, UninitializedFieldError};
+use std::collections::HashMap;
+
+#[derive(Builder)]
+#[builder(build_fn(error = "errors::YdbError"))]
+pub struct TopicOptions {
+    // Use TopicOptionsBuilder
+    #[builder(default)]
+    pub metering_mode: MeteringMode,
```

Comment:
What about use Option<MeteringMode> and None as unspecified? Will it more rust-native way?